config_examples/data-mappings.yml.example-synop-bufr

```diff
@@ -0,0 +1,17 @@
+data:
+    member_code3.center_id.data.core.weather.surface-based-observations.SYNOP:
```

Comment:
I would make clear that member_code3 is the ISO 3166 alpha-3 code.  Maybe call it ISO3C_country or similar.

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3

Reply:
ok I've changed this according to your suggestion